src/components/avatar/Avatar.js

```diff
@@ -55,21 +61,28 @@ class Avatar extends React.PureComponent<Props, State> {
     };
 
     render() {
-        const { avatarUrl, className, name, id, size = '' }: Props = this.props;
+        const { avatarUrl, className, name, id, size = '', isPending = false }: Props = this.props;
         const { hasImageErrored }: State = this.state;
         const classes = classNames(['avatar', className, { [`avatar--${size}`]: SIZES[size] }]);
 
+        const hasNoAvatar = !avatarUrl && !isPending;
+        const loadingBackgroundColor = lightGray;
+        const initialsBackgroundColor = `${getColor(id)}`;
+        const backgroundColor = hasNoAvatar || hasImageErrored ? initialsBackgroundColor : loadingBackgroundColor;
+
         let avatar;
-        if (avatarUrl && !hasImageErrored) {
-            avatar = <AvatarImage onError={this.onImageError} url={avatarUrl} />;
-        } else if (name) {
-            avatar = <AvatarInitials id={id} name={name} />;
+        if (!isPending && avatarUrl && !hasImageErrored) {
+            avatar = <AvatarImage onError={this.onImageError} url={`${avatarUrl}`} />;
+        } else if (!isPending && name) {
```

Comment:
If name is required when we don't have an avatar URL, can we skip the pending state and go straight to name if avatarUrl doesn't exist?

Reply:
The existing code says it is "required" in a comment but it still handled the UnknownUser case (where there is no name or avatarUrl). I left it the same way.